windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,36 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.1
+
+Issued:2021-06-23
+
+The WMCO 2.0.1 is now available with bug fixes. The components of the WMCO were released in [].
+
+[id="wmco-2-0-1-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+[id="wmco-2-0-1-image-timeout"]
+==== Increased image pull time-out duration
+
+Image pull time-out has been increased to 30 minutes.
+
+[id="wmco-2-0-1-bug-fixes"]
+=== Bug fixes
+
+* With the Windows `kube-proxy` component, when you create a LoadBalancer service, packets can be misrouted and reach an unintended destination. Now, you can set the `.status.loadBalancer.ingress.ip` to ensure packets are no longer wrongly routed to unintended destinations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**])
```

Comment:
Need to mention `on AWS` somewhere here as it is an AWS only issue.

>With the Windows `kube-proxy` component, when you create a LoadBalancer service, packets can be misrouted and reach an unintended destination.

This sentence makes it seem like this can still happen.